src/contract/table.ts

```diff
@@ -184,48 +187,19 @@ export class Table<RowType = any> {
      *  - `limit`: Maximum number of rows to return.
      * @returns {TableCursor<TableRow>} Promise resolving to a `TableCursor` of the table rows.
      */
-    first(maxRows: number, options: QueryOptions = {}): TableCursor<RowType> {
-        const tableRowsParams = {
-            table: this.name,
-            limit: maxRows,
-            code: this.account,
-            type: this.rowType,
-            scope: options.scope,
-        }
-
-        return new TableCursor<RowType>({
-            abi: this.abi,
-            client: this.client,
+    first(maxRows: number, params: QueryParams = {}): TableCursor<RowType> {
```

Comment:
Should we just get rid of this `first()` method like we talked about? I'm not sure it adds much value at this point since we can get the same with `.query({maxRows})`

Reply:
We can definitely - I wasn't sure if this still is useful as shorthand for that query though. No strong opinion.

Reply:
Yeah same here lol. I guess we can keep it around for now.